pkg/services/vimmachine.go

```diff
@@ -181,8 +182,8 @@ func (v *VimMachineService) GetHostInfo(c context.MachineContext) (string, error
 
 	vsphereVM := &infrav1.VSphereVM{}
 	if err := ctx.Client.Get(ctx, client.ObjectKey{
-		Namespace: ctx.Machine.Namespace,
-		Name:      ctx.Machine.Name,
+		Namespace: ctx.VSphereMachine.Namespace,
```

Comment:
nit: Does not matter what you use, the namespace for the entire hierarchy of CAPI/CAPV objects will be the same.

Reply:
ok, can be a good idea to `vspheremachine` and maintain the standard across calls. wdyt?